src/main/java/eu/enmeshed/client/EnmeshedClient.java

```diff
@@ -77,16 +74,17 @@ static EnmeshedClient configure(
   /*
    Attributes
   */
-  @RequestLine("GET /api/v2/Attributes?content.@type={0}&content.owner={1}&content.value.@type={2}")
```

Comment:
Why were these two query parameters `content.@type` and `content.owner` removed? If the removal is necessary, I would also expect a renumbering of the parameters.

Reply:
I renamed it